### ISSUE TITLE
I want the cron feature to work with more than just once per day

### ISSUE THREAD

**Issue by hansmosh** [nio-blocks/counter#22]
With cron you should be able to specify other intervals besides setting a time that it resets once per day. 

I don't actually know what way would make the most sense for this though. Change the cron properties form IntProperty to StringProperty to support values like "*" and "*/5"?

**Comment by mattdodge** [comment]
Yeah, that's exactly what we should do. I recall a TODO or something somewhere to a similar effect, but I can't find it.
